tensorrt_llm/_torch/configs/deepseek_v4.py

```
@@ -0,0 +1,105 @@
+        o_lora_rank=1024,
+        head_dim=512,
+        qk_rope_head_dim=64,
+        o_groups=8,
+        index_head_dim=128,
+        index_n_heads=64,
+        index_topk=512,
+        num_hash_layers=3,
+        hc_sinkhorn_iters=20,
+        hc_mult=4,
+        hc_eps=1e-06,
+        compress_ratios=[0, 0, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 128, 4, 0],
```

Ruff (0.15.11)

[warning] Do not use mutable data structures for argument defaults

Replace with `None`; initialize within function

(B006)